conda/recipes/libcudf_kafka/meta.yaml

```
@@ -23,10 +23,12 @@ build:
 
 requirements:
   build:
+    - python
     - cmake >=3.20.1
   host:
+    - python
     - libcudf {{version}}
-    - librdkafka >=1.6.0,<1.7.0a0
+    - librdkafka >=1.7.0,<1.8.0a0
   run:
     - {{ pin_compatible('librdkafka', max_pin='x.x') }} #TODO: librdkafka should be automatically included here by run_exports but is not
```

Would need to add `python` to `run` if it is needed in `host`

```suggestion
    - python
    - {{ pin_compatible('librdkafka', max_pin='x.x') }} #TODO: librdkafka should be automatically included here by run_exports but is not
```